src/message/headers/StreamMessageHeader.js

```diff
@@ -17,7 +17,6 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     flexDirection: 'row',
     backgroundColor: '#eee',
-    overflow: 'hidden',
```

Comment:
Removing 'overflow: hidden' doesn't seem to solve the problem for me anyway. Does it for you?